build-support/githooks/prepare-commit-msg

```diff
@@ -42,14 +42,6 @@ HAS_RUST_SKIP=$?
 grep "\[ci skip-build-wheels\]" "${COMMIT_MSG_FILEPATH}" > /dev/null
 HAS_WHEELS_SKIP=$?
 
-if [[ "${HAS_CI_SKIP}" -eq 1 ]] && [ "${NUM_NON_MD_FILES}" -eq 0 ]; then
-cat <<EOF >> "${COMMIT_MSG_FILEPATH}"
-
-# All CI will be skipped. Delete if not intended.
-[ci skip]
-EOF
-fi
```

Comment:
Unconsumed in GitHub config. Also we rarely update `.md` files anymore, so not really relevant.